wasmsdk/allocation.go

```diff
@@ -214,6 +214,42 @@ func lockWritePool(allocID, tokens, fee string) (string, error) {
 	return hash, err
 }
 
+func lockStakePool(providerType int, tokens, providerID, fee string) (string, error) {
```

Comment:
uint64 is supported on wasm, `ParseCoinStr` is unnecessary here.

Reply:
fixed in 6cc0793